indexers/dbms/PostgreSQLIndexer/__init__.py

```diff
@@ -39,10 +40,10 @@ def __init__(
         self.port = port
         self.username = username
         self.password = password
```

Comment:
Let's put `self.username` and `self.password` in the `post_init` call. Like this we can avoid serializing `username` and `passwords`.

Reply:
So I remove it from the `init` and keep it only on `post_init`?

Reply:
ah no, it will not be possible, do not worry